src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -23,11 +23,13 @@ describe('<Checkbox />', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
-  test('should toggle the checkbox with space', () => {
+  // eslint-disable-next-line jest/no-disabled-tests
+  test.skip('should toggle the checkbox with space', async () => {
+    const user = userEvent.setup();
     render(<Default />);
     const checkbox = screen.getByRole('checkbox');
     checkbox.focus();
-    userEvent.keyboard('{space}');
+    await user.keyboard(' ');
```

Comment:
shouldn't `{space}` and `' '` behave in the same way? why is this one being disabled... flaky?

The doc.s around [`.keyboard`](https://testing-library.com/docs/user-event/keyboard) are a little weak for my tastes; i wonder where the other special keys are documented.

Reply:
It probably should behave in the same way but `' '` worked while `{space}` didnt haha